WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -890,6 +890,37 @@ private extension NotificationsViewController {
         })
     }
 
+    /// Presents a confirmation action sheet for mark all as read action.
+    @objc func showMarkAllAsReadConfirmation() {
+        let title = NSLocalizedString(
+            "Mark all %1$@ notifications as read?",
+            comment: "Confirmation title for marking all notifications as read."
```

Comment:
Let's add an explanation to the `comment` indicating what the `%1$@` is for. Something like `%1$@ is a placeholder for the notification type.`